pkg/issuer/acme/dns/route53/route53.go

```diff
@@ -85,7 +62,7 @@ func (d *sessionProvider) GetSession() (*session.Session, error) {
 
 	useAmbientCredentials := d.Ambient && (d.AccessKeyID == "" && d.SecretAccessKey == "")
 
-	r := customRetryer{}
+	r := client.DefaultRetryer{}
```

Comment:
Should this be a NoOpRetryer instead? https://docs.aws.amazon.com/sdk-for-go/api/aws/client/#NoOpRetryer

Reply:
Oh, good point!

Reply:
If retries aren't done here at all, I think this block could be
```
	config := aws.NewConfig()
	sessionOpts := session.Options{
		Config: *config,
	}
```
right?

Reply:
@munnerz let me know if something else doesn't look right!